scripts/msys_build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir -p build && cd build || exit 1
+cmake ../ -G Ninja \
+    -DCMAKE_C_COMPILER=cl \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_PREFIX_PATH="$INSTALL_PREFIX;$INSTALL_PREFIX/include/librz;$INSTALL_PREFIX/include/librz/sdb" \
```

Comment:
Is this really necessary (the explicit `/include/...`)? Sounds more like something to be handled by `FindRizin.cmake`

Reply:
We do the same in Cutter but I can try to make it work

https://github.com/rizinorg/cutter/blob/9c7ef2465fd57c6ffa70126d0350b15d0ea1eb41/dist/CMakeLists.txt#L112